src/containerapp/azext_containerapp/tests/latest/test_containerapp_java_component.py

```diff
@@ -63,14 +64,47 @@ def test_containerapp_java_component(self, resource_group):
             JMESPathCheck('properties.componentType', "SpringBootAdmin"),
             JMESPathCheck('properties.ingress.fqdn', sba_name + "-azure-java.ext." + default_domain),
             JMESPathCheck('properties.provisioningState', "Succeeded"),
-                JMESPathCheck('properties.scale.minReplicas', 2),
-                JMESPathCheck('properties.scale.maxReplicas', 2)
+            JMESPathCheck('properties.scale.minReplicas', 2),
+            JMESPathCheck('properties.scale.maxReplicas', 2)
         ])
    
         # List Java Components
         java_component_list = self.cmd("containerapp env java-component list -g {} --environment {}".format(resource_group, env_name)).get_output_in_json()
         self.assertTrue(len(java_component_list) == 3)
 
+        route_yaml_text_create = f"""
+            springCloudGatewayRoutes:
+            - id: "route1"
+              uri: "https://otherjavacomponent.myenvironment.test.net"
+              predicates:
+              - "Path=/v1/path1"
+              - "After=2024-01-01T00:00:00.000-00:00[America/Denver]"
+              filters:
+              - "SetPath=/filter1"
+            - id: "route2"
+              uri: "https://otherjavacomponent.myenvironment.test.net"
+              predicates:
+              - "Path=/v2/path2"
+              - "After=2024-01-01T00:00:00.000-00:00[America/Denver]"
+              filters:
+              - "SetPath=/filter2"
+            """
+        route_yaml_name_create = f"{self._testMethodName}_route_create.yml"
+
+        write_test_file(route_yaml_name_create, route_yaml_text_create)
+        self.cmd("containerapp env java-component gateway-for-spring create -g {} -n {} --environment {} --route-yaml {}".format(resource_group, gateway_name, env_name, route_yaml_name_create), checks=[
+                JMESPathCheck('name', gateway_name),
+                JMESPathCheck('properties.componentType', "SpringCloudGateway"),
+                JMESPathCheck('properties.provisioningState', "Succeeded"),
+                JMESPathCheck('length(properties.springCloudGatewayRoutes)', 2),
```

Comment:
Please check more properties that you set, for example: `url`, `predicates`, `filters`

Reply:
Sure, add check in create command